src/table/src/TableRow.js

```diff
@@ -67,22 +72,32 @@ const TableRow = memo(
       onKeyPress(e)
     }
 
-    const themedClassName = theme.getRowClassName(appearance, intent)
+    const { className: themedClassName, ...boxProps } = useStyleConfig(
+      'TableRow',
+      { appearance, intent },
+      pseudoSelectors,
+      {
+        display: 'flex'
+      }
+    )
+
+    const { height: themeHeight, ...restBoxProps } = boxProps
```

Comment:
Might as well pull this out at the same time as we do `themedClassName` above to avoid the cost of 2 spreads.